src/lint/index.ts

```diff
@@ -14,6 +14,11 @@ export const profilePath = path.join(
   "../../resources/lint/profiles"
 );
 
+const profiles = fs
```

Comment:
This can work in the short term, but I don't love it for a long term solution.

We should have a method for registering new profiles to the linter that makes sense.  While this works I just don't see it as scalable. I think someone should investigate a better way of doing this in the future.